corehq/apps/app_manager/static/app_manager/js/manage_releases.js

```diff
@@ -0,0 +1,101 @@
+hqDefine('app_manager/js/manage_releases', [
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+    'hqwebapp/js/widgets_v4', // using select2/dist/js/select2.full.min for ko-select2 on location select
+], function (
+    $,
+    ko,
+    _,
+    initialPageData
+) {
+    'use strict';
+    $(function () {
+        var enabledAppRelease = function (details) {
```

Comment:
`details` has a bunch of properties; it'd be good to verify they're the correct properties with [assert_properties](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/assert_properties.js)